### PR TITLE
Add Test: #25 0903001f07b4d0adf7c88b59ba441be83fcdc75e

### PR DESCRIPTION
**Pull request by tkoyama010**
Hi. I added test for #25 0903001f07b4d0adf7c88b59ba441be83fcdc75e . Thank you for reading.

### CODE REVIEW

File: tests/special/test_spherical_bessel_yn.f90 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  use utils, only: assert
+  use special, only: spherical_bessel_yn
+  implicit none
+
+  ! check a few value against the ones computed with SciPy:
+  ! note we use a rather 'large' eps - SciPy computes some the values we compare
+  ! to with CEPHES whereas we use AMOS
+
+  integer :: i
+  real(dp), parameter :: eps = 1d-13
```
Comment (certik) [inline tests/special/test_spherical_bessel_yn.f90]:
```suggestion
  real(dp), parameter :: eps = 1e-13_dp
```

**Comment by certik** [comment]
Thanks @tkoyama010 ! +1 to merge. I just provided a very minor suggestion.

**Comment by tkoyama010** [comment]
Thanks for your review!

**Review by certik — APPROVED** [review]

**Comment by certik** [comment]
Thanks @tkoyama010.

**Pull request merged**